guide/daily-spending-wallet/backup-and-recovery/manual-backup.md

```diff
@@ -134,179 +109,208 @@ Guides and gives users actionable items on how to safely **handle** their recove
+   layout = "shadow"
+   caption = "This UI will be very different depending on the cloud provider and what permissions are required."
+   alt-text = "Screen of placeholder where cloud provider UI would be"
+   width = 250
+   height = 541
+%}
 
-</div>
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase-intro.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase-intro@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase-intro@2x.png"
+   layout = "shadow"
+   caption = "Explain what a recovery phrase is."
+   alt-text = "Screen with a description of what a recovery phrase is"
+   width = 250
+   height = 541
+%}
 
-<div class="center" markdown="1">
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-next-steps.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-next-steps@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-next-steps@2x.png"
+   layout = "shadow"
+   caption = "Explain what is about to happen and what the user should do."
+   alt-text = "Screen explaining that the recovery phrase will be shown next and how to write it down"
+   width = 250
+   height = 541
+%}
 
-{% include image.html
-   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/copy-recovery-phrase-reveal.png"
-   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/copy-recovery-phrase-reveal@2x.png"
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase@2x.png"
+   layout = "shadow"
+   caption = "Displays the recovery phrase with numbering."
+   alt-text = "Screen displaying a 12 word recovery phrase"
    width = 250
-   height = 600
-   alt-text = "Reveal option where words are not readable by default"
-   caption = "Hold down to reveal UI from the [UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1230%3A223)"
-   layout = "float-right-desktop -background -shadow"
+   height = 541
 %}
 
-**Option 3: Using Interactions to “Uncover” or “Reveal”**
+</div>
 
-A clever way to give a user their recovery phrase is by creating an interaction where they have to “uncover” it. This can entail hovering over a blurred word to reveal it, or holding down a button to show it.
+### Print template
```

Comment:
Should we link out to this [Figma community file](https://www.figma.com/community/file/1110806582648546839) that contains a template? We could combine both of our designs in there.

Reply:
There's no point combining them, I'll just use yours. It might be nice to have a descriptor in the community file but a) we can do that later and b) that might be something we ask for developer consultation on.

Reply:
Done